Makefile

```diff
@@ -45,3 +45,18 @@ test:
 
 testcov:
 	py.test --cov=ideascube/ --cov-report=term-missing --migrations
+
+test-data-migration:
+	set -e ; \
+	BRANCH=$$(git rev-parse --abbrev-ref HEAD); \
+	LATEST_TAG=$$(git describe --abbrev=0 --tags); \
+	\
+	echo "# Loading some data at $$LATEST_TAG"; \
+	git checkout $$LATEST_TAG; \
+	rm -fr storage; \
+	make migrate; \
+	python manage.py loaddata --database=default test-data/data.json; \
```

Comment:
> Unless you meant for us to test reindexation in the CI as well, which is an interesting thought on its own. /me goes and ponder on this a bit...

This is what I meant. We had problems in the past with index and migration. I would prefer to test it also. (But, I're right, it will make the CI jobs longer. I don't know how many time it add to the job to reindex the content)